llvm/lib/Target/AMDGPU/AMDGPU.td

```diff
@@ -743,6 +743,12 @@ def FeatureAtomicGlobalPkAddBF16Inst : SubtargetFeature<"atomic-global-pk-add-bf
  [FeatureFlatGlobalInsts]
 >;
 
+def FeatureAtomicBufferPkAddBF16Inst : SubtargetFeature<"atomic-buffer-pk-add-bf16-inst",
```

Comment:
I believe it is above FeatureAtomicGlobalPkAddBF16Instin downstream. Can you fix the order here or there?

Reply:
This doesn't exist downstream

Reply:
Fixed